### PR TITLE
[10.0][ADD] Module Sale Merge Draft Invoice

### PR DESCRIPTION
**Pull request by rgarnau**
This module merges new invoices with existing draft ones when creating them if partner and currency are the same. It depends on #535 

@jbeficent @etobella

### CODE REVIEW

File: sale_merge_draft_invoice/README.rst [hunk 1]
```diff
@@ -0,0 +1,90 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :alt: License LGPL-3
```
Comment (MiquelRForgeFlow) [inline sale_merge_draft_invoice/README.rst]: Some files of this module are AGPL and others LGPL. Please, maintain full consistency to one option.

**Comment by rgarnau** [comment]
@mreficent done. Can you please update your review?

File: sale_merge_draft_invoice/README.rst (hunk 1 above)
Comment (MiquelRForgeFlow) [reply sale_merge_draft_invoice/README.rst]: :target: https://www.gnu.org/licenses/lgpl

Comment (rgarnau) [reply sale_merge_draft_invoice/README.rst]: added!

File: sale_merge_draft_invoice/README.rst [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+This module merges new invoices with existing draft ones when creating them.
+
+The merge between invoices is done when the partner and the currency are the
+same.
+
+This feature is activated from Sales / Settings and assigned to the company
+settings.
+
+Users assigned to the group 'Change sale invoice merge proposal'
```
Comment (JordiBForgeFlow) [inline sale_merge_draft_invoice/README.rst]: Can you rename the group to 'Change sale to invoice merge proposal'.

File: sale_merge_draft_invoice/README.rst [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+have the possibility to activate or deactivate this option only when creating a
+single invoice from a sale order.
+
+Configuration
+=============
+
+#. Go to 'Sales / Configuration / Settings'
+#. In 'Quotations & Sales' go to 'Invoices' and activate the option 'Merge
+   new invoices with existing draft ones'.
+#. Go to 'Settings / Users / Groups' and assign to the group 'Change sale
```
Comment (JordiBForgeFlow) [inline sale_merge_draft_invoice/README.rst:27]: 'Change sale to invoice merge proposal'.

File: sale_merge_draft_invoice/README.rst [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Jordi Ballester Alomar <jordi.ballester@eficent.com>
```
Comment (JordiBForgeFlow) [inline sale_merge_draft_invoice/README.rst]: Put yourself first! You deserve it.

File: sale_merge_draft_invoice/model/res_company.py [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
```
Comment (JordiBForgeFlow) [inline sale_merge_draft_invoice/model/res_company.py]: put https instead of http everywhere

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Technical and functional review. Minor comments

**Comment by JordiBForgeFlow** [comment]
@rgarnau When you want to invoice multiple sales orders at once, the checkbox 'Merge with draft invoices' is not visible, and should be.

**Comment by rgarnau** [comment]
@jbeficent @mreficent - Both reviews have been attended.

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by etobella — APPROVED** [review]
LGTM :+1:

**Pull request merged**